app/components/ak-breadcrumbs/container/index.hbs

```diff
@@ -0,0 +1,14 @@
+<AkStack
+  ...attributes
+  id={{this.id}}
+  @direction={{@direction}}
+  @alignItems={{@alignItems}}
+  @flexWrap={{@flexWrap}}
+  @justifyContent={{@justifyContent}}
+  @width={{@width}}
+  @tag={{'ul'}}
+  @spacing={{@spacing}}
+  {{breadcrumbs-container id=this.id}}
+>
+  {{yield}}
```

Comment:
Do we require this yield ? because it is not designed to work child blocks

Reply:
Yeah. Yielding is required if one wished to use the breadcrumbs in this format.

```
<AkBreadcrumbs::Container>
  <AkBreadcrumbs::Item />
  <AkBreadcrumbs::Item />
</AkBreadcrumbs::Container>
```